src/networking.c

```diff
@@ -683,6 +683,16 @@ void addReplyDouble(client *c, double d) {
     }
 }
 
+void addReplyBigNum(client *c, const char* num, size_t len) {
```

Comment:
let's change the code in DEBUG PROTOCOL to use it.
and let's extend redis-cli too

Reply:
@oranagra , it requires updating the hiredis, seems like our current version do not have it: https://github.com/redis/redis/blob/ac8b1df8850cc80fbf9ce8c2fbde0c1d3a1b4e91/deps/hiredis/read.c#L528
While on hiredis master it exists:
https://github.com/redis/hiredis/blob/f5f31ff9b92b6bdf628716449d0d0782ceb7704a/read.c#L614

Do you want to do it on this PR or on another PR?

Reply:
Let's do it in another PR. 
Need to look at the last PR that updated hiredis and see the methodology (git subtree) and also chose the right hiredis version